spec/models/store_spec.rb

```diff
@@ -1,9 +1,24 @@
 require 'rails_helper'
 
 RSpec.describe Store do
+  describe ".class_methods" do
```

Comment:
Wouldn't need to test this method since we are assuming enums given class method Store.active works.